2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,65 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+const recipes = {
+  first: {
+    title: "Spaghetti",
+    serves: 6,
+    ingredients: ["pasta", "mince", "tomato"],
+  },
+  second: {
+    title: "Hummus",
+    serves: 2,
+    ingredients: ["Tahini", "chickpea"],
+  },
+  third: {
+    title: "fish & chips",
+    serves: 4,
+    ingredients: ["fish", "potato", "bread crumbs"],
+  },
+  forth: {
+    title: "porridge",
+    serves: 1,
+    ingredients: ["oat", "milk"],
+  },
+  fifth: {
+    title: "guacamole",
+    serves: 3,
+    ingredients: ["avocado", "tomato", "onion", "parsley"],
```

Comment:
You have 4 ingredients here. What data structure are they stored in?
You also have 5 recipes. What kind of data structure do you think your recipes should be stored in?
The code below this comment is very repetitive, what Javascript APIs have you used in the past few weeks that would help reduce the repetition?